### DISCUSSION TITLE
Is rebinding Mouse keys intended?

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#16132] [Q&A] [answered]
Talked about here: https://www.youtube.com/watch?v=Q0Jq70OV2BA

It allows the player to press 4 keys (on standard) but IIRC, It's only functional for 2 keys. Right?
This feels like that [binding scroll wheel issue #12339](https://github.com/ppy/osu/issues/12339) but less critical.

**Comment by peppy** [comment, marked as answer]
Yes, this is intended behaviour.
